pkg/ingester/locker.go

```diff
@@ -30,7 +30,7 @@ type paddedMutex struct {
 // the same mutex twice).
 type fingerprintLocker struct {
 	fpMtxs    []paddedMutex
-	numFpMtxs uint
+	numFpMtxs uint32
```

Comment:
Does this really make enough difference? Just curious if it does.

Reply:
This makes no difference.  its there as I pulled in the fnv32 implementation to avoid copying a string to a []byte, and reused it here.